core/services/bulletprooftxmanager/eth_confirmer.go

```diff
@@ -851,7 +852,7 @@ func saveSentAttempt(db *gorm.DB, attempt *models.EthTxAttempt) error {
 	}
 	attempt.State = models.EthTxAttemptBroadcast
 	return postgres.GormTransaction(context.Background(), db, func(tx *gorm.DB) error {
-		if err := tx.Exec(`UPDATE eth_txes SET broadcast_at = NOW() WHERE id = ? AND broadcast_at IS NULL`, attempt.EthTxID).Error; err != nil {
+		if err := tx.Exec(`UPDATE eth_txes SET broadcast_at = NOW() WHERE id = ? AND broadcast_at < NOW()`, attempt.EthTxID).Error; err != nil {
```

Comment:
can you elaborate on this change - how do we know broadcast_at is now always set?

Reply:
Good catch, we don't. Will add a null check

Reply:
It was actually wrong before. Leftover cruft from a path I decided not to take in the previous PR for nonce syncing